### PR TITLE
Fixes for Julia 1.0 (fixes issue #11)

### PR DESCRIPTION
**Pull request by DilumAluthge**
Fixes for Julia 1.0 (fixes issue #11)

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1 +1 @@
-julia 0.7-
+julia 1.0
```
Comment (joshday) [inline REQUIRE]: I think this should be `julia 0.7` to aid people transitioning from 0.6 to 1.0.

**Comment by joshday** [comment]
If I can be greedy, how about adding 0.7 and 1.0 to the travis build here:

https://github.com/JuliaComputing/PooledArrays.jl/blob/4f7206e527ffd77a81bce73567d4e54f11644d37/.travis.yml#L6

**Comment by DilumAluthge** [comment]
@joshday done and done

**Comment by joshday** [comment]
Awesome, thanks!  I'll merge when CI goes green.

**Comment by DilumAluthge** [comment]
Thank you @joshday! When you get a chance, can you tag a new release?

**Pull request merged**